### PR TITLE
Tech task: Docker updates

### PR DESCRIPTION
**Pull request by jhanggi**
# Release Notes

Tech task: improve docker setup including tags and improved database/secrets templates.

## Additional Context

This is mostly pulled up from the work we're doing for Dartmouth around getting the app ready for Kubernetes https://github.com/tablexi/nucore-dartmouth/pull/306

### CODE REVIEW

File: config/database.yml.oracle.template [hunk 1]
```diff
@@ -1,10 +1,11 @@
 defaults: &defaults
   adapter: oracle_enhanced
   database: //localhost:1521/ORCLCDB.localdomain
+  database: //<%= ENV.fetch("ORACLE_HOST", "localhost") %>:<%= ENV.fetch("ORACLE_PORT", 1521) %>/<%= ENV.fetch("ORACLE_SERVICE_NAME", "ORCLCDB.localdomain") %>
   # Alternative format:
-  # database: (DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=localhost)(PORT=1521))(CONNECT_DATA=(SERVER=DEDICATED)(SERVICE_NAME=ORCLCDB.localdomain)))
+  # database: (DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=<localhost)(PORT=1521))(CONNECT_DATA=(SERVER=DEDICATED)(SERVICE_NAME=ORCLCDB.localdomain)))
```
Comment (giladshanan) [inline config/database.yml.oracle.template:6]: What is this `<` doing?

Comment (jhanggi) [reply config/database.yml.oracle.template:6]: Oops. That probably shouldn't be there.

Comment (jhanggi) [inline config/database.yml.oracle.template:6]:
```suggestion
  # database: (DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=localhost)(PORT=1521))(CONNECT_DATA=(SERVER=DEDICATED)(SERVICE_NAME=ORCLCDB.localdomain)))
```

File: config/database.yml.oracle.template [hunk 2]
```diff
@@ -18,3 +19,7 @@ development:
 test:
   <<: *defaults
   username: "c##nucore_nu_test"
+
+production:
+  <<: *defaults
+  database: <%= ENV.fetch("MYSQL_DATABASE", "") %>
```
Comment (jhanggi) [inline config/database.yml.oracle.template:25]: Probably should change to `ORACLE_USERNAME`

**Pull request merged**